kubetest/extract_k8s.go

```diff
@@ -346,9 +346,30 @@ func setReleaseFromHTTP(prefix, suffix string, getSrc bool) error {
 	return getKube(url, strings.TrimSpace(string(release)), getSrc)
 }
 
-func setupGciVars(family string) (string, error) {
-	p := "container-vm-image-staging"
-	b, err := control.Output(exec.Command("gcloud", "compute", "images", "describe-from-family", family, fmt.Sprintf("--project=%v", p), "--format=value(name)"))
+var parseGciExtractOption = func(option string) (string, map[string]string) {
+	arr := strings.Split(option, "?")
+	family := arr[0]
+	paramsMap := map[string]string{
+		// default values
+		"project":            "container-vm-image-staging",
+		"version-map-bucket": "container-vm-image-staging",
+	}
+	if len(arr) == 2 {
+		params := strings.Split(arr[1], ":")
+		for _, param := range params {
+			kv := strings.Split(param, "=")
+			paramsMap[kv[0]] = kv[1]
+		}
+	}
+	return family, paramsMap
+}
+
+var gcloudGetImageName = func(family string, project string) ([]byte, error) {
+	return control.Output(exec.Command("gcloud", "compute", "images", "describe-from-family", family, fmt.Sprintf("--project=%v", project), "--format=value(name)"))
+}
+
+func setupGciVars(family string, p string) (string, error) {
```

Comment:
either family string, project string 
or f, p string

Reply:
Change to f, p string.